src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
✏️ Do we need this file? Should we wait to commit it until we need it?

Reply:
It was generated and I'm not against removing it.

Reply:
Generated by what?

Reply:
I vote remove until we need it.